google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -2365,6 +2582,19 @@ class StorageObjectIterator(resource_iter_class_factory(
             StorageBucketIterator,
         ]}),
 
+    'kubernetes_cluster': ResourceFactory({
+        'dependsOn': ['project'],
+        'cls': KubernetesCluster,
+        'contains': [
+            KubernetesNodeIterator,
```

Comment:
Look back at the k8 document.

The cluster contains Node, Namespace, ClusterRole and ClusterRoleBinding.

The Namespace contains Pod, Role and RoleBinding.

Reply:
Updated.